ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -45,6 +48,7 @@
   private static final String USER_AGENT_HEADER_NAME = "user-agent";
 
   private final OkHttpClient mClient;
+  private final CookieManager mCookieManager;
```

Comment:
You can skip this field and just construct in inline. `mClient.setCookieHandler(new CookieManager());`